src/sage/combinat/cluster_algebra_quiver/cluster_seed.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+# sage.doctest - sage.graphs sage.modules
```

Comment:
Here it is `# sage.doctest - sage.graphs sage.modules` but in file `src/sage/combinat/cluster_algebra_quiver/mutation_class.py` it is `# sage.doctest: optional - sage.graphs`. What's the correct form ?

Reply:
It's a typo here, thanks for catching it